client/src/components/FehlerErstellen.js

```diff
@@ -15,6 +19,13 @@ const Fehlererstellen = () => {
   const handleSubmit = (e) => {
     //e.preventDefault();
 
+    toast.success("Fehler erstellt.", {
```

Comment:
nur aufrufen wenn fehler wirklich erstellt wurde also wenn in der response der "error = false" ist